extensions/rename.md

```diff
@@ -0,0 +1,93 @@
+---
+title: IRCv3 `rename` Extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Peter Powell"
+    email: "petpow@saberuk.com"
+    period: "2017"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `rename` capability name. Instead, implementations SHOULD use the
+`draft/rename` capability name to be interoperable with other software
+implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+## Introduction
+
+Currently there is no reasonable way to rename channels. All existing methods
+involve closing down the channel and redirecting it to the a new one. This
+is undesirable as access and ban lists needs to be manually copied. It also
+causes problems for client logging as history is split across multiple files.
+
+This specification introduces a method of natively renaming channels that
+does not suffer from these faults.
+
+## Architecture
+
+### Capabilities
+
+This specification introduces the `draft/rename` capability. This capability
+informs the server that the client is capable of handling `RENAME` messages.
```

Comment:
https://github.com/ircv3/ircv3-specifications/pull/292/files#diff-8397424fc8fee1b5d347e97c8787b2fbR14 I can't find the comment where it is initially implemented/suggested